### ISSUE TITLE
SA1117 applying to SuppressMessage

### ISSUE THREAD

**Issue by GregReddick** [DotNetAnalyzers/StyleCopAnalyzers#2263]
In In 1.1.0-beta001, SuppressMessage attributes trigger SA1117. This was not the case in the original StyleCop or 1.0.0 of StyleCopAnalyzers. Example of what used to not trigger this warning:

```
[SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "long",
	Justification = "Not a data type")]
public static int GetLongCountDigit(TimeSpanX mayaDate, int index)
```

**Comment by sharwell** [comment]
Duplicate of #1712. :smile:
